float64[] t
convex_mpc_controller/states states
convex_mpc_controller/inputs inputs
bool solved